tests/bats/Makefile

```diff
@@ -0,0 +1,71 @@
+#  SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+include $(CURDIR)/versions.mk
+include $(CURDIR)/common.mk
+
+BATS_IMAGE = batstests:$(GIT_COMMIT_SHORT)
+
+TEST_CHART_REPO ?= "oci://ghcr.io/nvidia/k8s-dra-driver-gpu"
+TEST_CHART_VERSION ?= $(VERSION_GHCR_CHART)
+TEST_CHART_LASTSTABLE_REPO ?= "oci://ghcr.io/nvidia/k8s-dra-driver-gpu"
+TEST_CHART_LASTSTABLE_VERSION ?= "25.3.2-2c250af3-chart"
+TEST_NVIDIA_DRIVER_ROOT ?= "/run/nvidia/driver"
+
+# Currently consumed in upgrade test via
+# kubectl apply -f <URL> (can be a branch, tag, or commit)
+TEST_CRD_UPGRADE_TARGET_GIT_REF ?= $(GIT_COMMIT_SHORT)
+
+
```

Comment:
Can we add one more phone target here as:

```
default: bats-tests
```

That way you can invoke this without any target and it runs the `bats-tests` target.

Reply:
> one more phony target 

and this brings me back to the question that I looked up an answer for a couple of times, and then forgot soon after: when to add `.PHONY`? (no need to answer that here :).